include/aspect/introspection.h

```diff
@@ -51,23 +56,33 @@ namespace aspect
    * @ingroup Simulator
    */
   template <int dim>
-  struct Introspection
+  struct Introspection: public dealii::FEVariableSystem<dim>
```

Comment:
Ah, do you really want to put this class into namespace `dealii`? That would seem awkward and a violation of the encapsulation principle. If we ever move it into deal.II itself, we can always clone it and just deprecate the old version.

I'm slightly sad that the nice and self-contained `Introspection` class now consists of parts that one has to look up on different pages. But I guess that's what it is. Unless you want to integrate the FEVariableSystem (which at this point doesn't contain very many things any more) back into `Introspection`.